tests/functional/p/protected_access.py

```diff
@@ -27,3 +27,17 @@ def nargs(self):
 class Application(metaclass=MC):
     def __no_special__(cls):
         nargs = obj._nargs  # [protected-access]
+
+
+class Light:
+    @property
+    def _light_internal(self) -> set[str]:
+        return set()
```

Comment:
```suggestion
    def _light_internal(self):
        return 42
```

`set[str]` requires Python 3.9. Best to just remove it.